crates/gateway-types/src/reply.rs

```diff
@@ -187,17 +187,18 @@ pub mod call {
 
 /// Used to deserialize replies to Starknet transaction requests.
 ///
-/// We only care about the statuses so we ignore other fields.
 /// Please note that this does not have to be backwards compatible:
 /// since we only ever use it to deserialize replies from the Starknet
 /// feeder gateway.
 #[serde_as]
-#[derive(Clone, Copy, Debug, Deserialize, PartialEq, Eq)]
+#[derive(Clone, Debug, Deserialize, PartialEq, Eq)]
 pub struct TransactionStatus {
```

Comment:
Some field names do not seem to match what sepolia gives rn:
```
tx_status
tx_failure_reason
tx_revert_reason
```

for example:
https://alpha-sepolia.starknet.io/feeder_gateway/get_transaction_status?transactionHash=0x4fef839b57a7ac72c8738dc821897cc605b5cc5aafa487e445e9282ac37ac23
https://alpha-sepolia.starknet.io/feeder_gateway/get_transaction_status?transactionHash=0x063695761faf48340732db8562654cabce9b7a2c22d436f47d5489f52f28a5a6

Reply:
sorry, I didn't notice :thinking: